extensionsGallery-insider.json

```diff
@@ -2413,7 +2413,7 @@
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.7.md"
+									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.8.md"
```

Comment:
do you need to keep all versions of the readme file? if not you can simply have one file and make the version update easier.

Reply:
No, probably not. I can address this next time we update the extension.